components/bl-skeleton/src/index.js

```diff
@@ -10,7 +14,7 @@ export default function Skeleton({ component }) {
   return (
     <div
       className={ cn('bl-customComponent-skeleton',  variant, animation, classList) }
-      style={{ ...style, width: width || '100%', height: height || '100%' }}>
+      style={{ ...style, width: width || '100%', height: height || '100%', margin: validMargin }}>
```

Comment:
I assume `Styles Logic` should rewrite all styles from the settings, so `style` should be at the end.

Reply:
@StasivPavlo conflicts must be resolved